core/lib/generators/spree/install/install_generator.rb

```diff
@@ -79,35 +79,40 @@ def create_overrides_directory
     end
 
     def configure_application
-      application <<-APP
-
-    config.to_prepare do
-      # Load application's model / class decorators
-      Dir.glob(File.join(File.dirname(__FILE__), "../app/**/*_decorator*.rb")) do |c|
-        require_dependency(c)
+      application <<-RUBY
```

Comment:
I'm ok with keeping these changes (both commits for this file) but maybe now it's better to move them into a separate PR. They don't seem to be related to this one anymore, right?

Reply:
Done ✅